code/game/gamemodes/vampire/vampire_powers.dm

```diff
@@ -265,20 +265,28 @@
 	for(var/mob/living/carbon/C in hearers(4))
 		if(C == user)
 			continue
+		if(!affects(C))
+			continue
 		if(ishuman(C))
 			var/mob/living/carbon/human/H = C
 			if(H.check_ear_prot() >= HEARING_PROTECTION_TOTAL)
 				continue
-		if(!affects(C))
-			continue
-		to_chat(C, "<span class='warning'><font size='3'><b>You hear a ear piercing shriek and your senses dull!</font></b></span>")
+			if(H.check_ear_prot() >= HEARING_PROTECTION_MINOR)
+				to_chat(C, "<span class='warning'><font size='3'><b>You hear a dull shriek and instinctively cover your ears!</font></b></span>")
+				C.Confused(10)
+				C.Stuttering(10)
+				C.Jitter(50)
+				continue
+		to_chat(C, "<span class='warning'><font size='3'><b>You hear an ear piercing shriek and your senses dull!</font></b></span>")
 		C.Weaken(4)
 		C.AdjustEarDamage(0, 20)
 		C.Stuttering(20)
 		C.Stun(4)
 		C.Jitter(150)
 	for(var/obj/structure/window/W in view(4))
 		W.deconstruct(FALSE)
+	for(var/obj/machinery/light/L in view(6))
+		L.deconstruct(FALSE)
```

Comment:
Another opinion, but I think that instead of the whole light being destroyed, only the bulb should be shattered, using the `break_light_tube()` proc.

Reply:
Agreed, switched to the suggest light-smashing method- Thanks Steel, much more elegant !